spec/requests/providers_spec.rb

```diff
@@ -464,6 +464,22 @@ def have_endpoint_attributes(expected_hash)
     end
   end
 
+  describe "Providers validate" do
```

Comment:
The action is `verify_credentials`, so should this say:?
```suggestion
  describe "Providers verify_credentials" do
```